### DISCUSSION TITLE
Tailscale fail to install on exsisting container

### DISCUSSION THREAD

**Discussion by DJKatastrof** [tteck/Proxmox#1967] [Help Needed] [answered]
Any ideas?

![image](https://github.com/tteck/Proxmox/assets/64254359/1e8eab04-bf1b-46c8-92d1-815546d36480)

**Comment by tteck** [comment]
Proxmox shell or SSH?

**Reply by tteck** [reply, marked as answer]
> To Install Talescale on an existing LXC, run the command below in the Proxmox VE Shell.

Unless your node is named "tailsc", you didn't execute the script in the Proxmox shell.

```
  ______      _ __                __        
 /_  __/___ _(_) /_____________ _/ /__      
  / / / __ `/ / / ___/ ___/ __ `/ / _ \     
 / / / /_/ / / (__  ) /__/ /_/ / /  __/     
/_/  \__,_/_/_/____/\___/\__,_/_/\___/      
                                            
This will add Tailscale to an existing LXC Container ONLY. Proceed(y/n)?y
Installing Tailscale...
Installed Tailscale
 ✔ Completed Successfully!
 Reboot 111 LXC to apply the changes, then run tailscale up in the LXC console
```

**Comment by DJKatastrof** [comment]
Hi!

It was in the promox shell.

________________________________
Från: tteckster ***@***.***>
Skickat: Friday, October 27, 2023 2:00:58 PM
Till: tteck/Proxmox ***@***.***>
Kopia: DJKatastrof ***@***.***>; Author ***@***.***>
Ämne: Re: [tteck/Proxmox] Tailscale fail to install on exsisting container (Discussion #1967)


To Install Talescale on an existing LXC, run the command below in the Proxmox VE Shell.

Unless your node is named "tailsc", you didn't execute the script in the Proxmox shell.

  ______      _ __                __
 /_  __/___ _(_) /_____________ _/ /__
  / / / __ `/ / / ___/ ___/ __ `/ / _ \
 / / / /_/ / / (__  ) /__/ /_/ / /  __/
/_/  \__,_/_/_/____/\___/\__,_/_/\___/

This will add Tailscale to an existing LXC Container ONLY. Proceed(y/n)?y
Installing Tailscale...
Installed Tailscale
 ✔ Completed Successfully!
 Reboot 111 LXC to apply the changes, then run tailscale up in the LXC console

**Reply by DJKatastrof** [reply]
Wait. 
Sorry. It was in the container. Will do in the promox shell

**Comment by danielo515** [comment]
So you need an already existing container (maybe a ubuntu one) and this script will ask you wich container to install to?

**Reply by tteck** [reply]
yes

![Screenshot 2024-02-06 11 04 20 AM](https://github.com/tteck/Proxmox/assets/21342670/3f356d39-4215-4bf5-b43e-9349baabfbc6)

**Reply by danielo515** [reply]
Awesome, thanks

**Reply by Proemis** [reply]
I tried using the helper scrips to install a [ Ubuntu 24.04 LXC ](https://tteck.github.io/Proxmox/?id=Ubuntu)(and then followed with the CloudInit setup). I then ran the script for Tailscale but it doesn’t see the Ubuntu LXC (but does see my Scrypted LXC and Docker as per screenshot).

Do you have any ideas on what to do so the Tailscale helper script finds the Ubuntu LXC?
I tried rebooting the Ubuntu LXC but no luck.
<img width="490" alt="Screenshot 2024-08-30 at 12 14 52 PM" src="https://github.com/user-attachments/assets/c252853a-0ffa-4647-881a-d9232443389e">
Thanks.
